WatchFaceAlphaKotlin/build.gradle

```diff
@@ -17,26 +17,26 @@
 buildscript {
     ext {
         // SDK and tools
-        kotlin_version = '1.4.20'
+        kotlin_version = '1.5.0'
 
-        watchFaceAlphaVersion = '1.0.0-alpha04'
+        watchFaceAlphaVersion = '1.0.0-alpha17'
         activityVersion = '1.1.0'
         appCompatVersion = '1.2.0'
         constraintLayoutVersion = '2.0.2'
         coreTestingVersion = '2.1.0'
         coroutines = '1.4.2'
-        lifecycleVersion = '2.2.0'
+        lifecycleVersion = '2.3.1'
         materialVersion = '1.2.1'
-        roomVersion = '2.2.5'
+        roomVersion = '2.3.0'
```

Comment:
Nit: There's a few versions here that are no longer being used that'd be good to clean up.

Reply:
Done.